src/services/array-functions.js

```diff
@@ -14,7 +19,14 @@ export function map(theArray, fnc){
 //fnc will return true or false, if true add the item to the new array else do not
 //return the new array
 export function filter(theArray, fnc){
-
+  let filteredArray = [];
+  for(var i = 0; i < theArray.length; i++){
+    // let item = fnc(theArray[i]);
+    if (theArray[i].length === 3) {
```

Comment:
Double check this part. You are doing the correct thing in your comment on line 24. 
In this case, 'item' will be either true or false
use that to determine whether or not to push to the array. 
we can't hard code length === 3, filter is not suppose to know what we are checking for